libs/product/src/models/composite-product-item.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffProduct } from './product';
+
+export interface DaffCompositeProductItem {
+	id: number;
+	required: boolean;
+	title: string;
+	type: string;
+	options: DaffCompositeProductItemOption[];
+}
+
+export interface DaffCompositeProductItemOption extends DaffProduct {
+	can_change_quantity: boolean;
+	id: string;
```

Comment:
number | string

Reply:
this change causes a lot of errors because it extends DaffProduct. I'll make this change in a different PR.